### ISSUE TITLE
Help me choose a GPU 

### ISSUE THREAD

**Issue by Huuan007** [w-okada/voice-changer#1206]
Hello, do you have any recommendations for me to choose one of the following graphics cards

1. Rtx 3060 [Msi Gaming X](342 USD)
2. Rtx 4060 ti [Msi Gaming X] 8gb ver (435-491 USD) 
3. Rtx 4060 ti [Msi Gaming X] 16gb ver (560 USD) 
Note:IDR to USD

If there is no significant delay, voice input and output, and no glitching or chopped voice between 3060 and 4060ti 8gb, I will buy 3060. However, I need advice from you if my choice is right or not. 

By the way, these are my PC specs:
CPU: Ryzen 5 5600G (stock Cooler) 
Motherboard: Asus TUF Gaming B550 plus wifi II
RAM: t-force delta 16gb ddr4 3200mhz cl16 (dual channel) 
Ssd: Kingston Kc3000 1TB
Psu: Cooler master g600 80+ gold

**Comment by KitsuneShan** [comment]
There will be differences because the percentage of the GPU used won't be the same between these cards. Usually you aim for the one with the most cuda cores so, in this case, the 4060Ti may be the best option.
But keep in mind that even then it may not be enough to run it within certain demanding games (if that's what you aim to do).
I have a 3080Ti and there are cases where certain games or specific points of a game that requires more performance makes RVC to increase the delay of the output. A friend have a 3080 (not Ti) and she runs into choppy cuts and stutter when on similar scenarios.
So if you want to use RVC without much issues, there is actually not "too much" GPU option. VRAM it's very important too so get the 16GB model.

Also consider that in a couple of months nvidia will launch the 5000 series. While it will be just the highest top ends and not the mid range ones, I am sure that it will also make every other card to drop in price. So you may want to hold a little bit.

**Comment by fabio1shot** [comment]
I Will Suggest you 3060

I have a question does CPU matter ?

**Comment by Huuan007** [comment]
> There will be differences because the percentage of the GPU used won't be the same between these cards. Usually you aim for the one with the most cuda cores so, in this case, the 4060Ti may be the best option. But keep in mind that even then it may not be enough to run it within certain demanding games (if that's what you aim to do). I have a 3080Ti and there are cases where certain games or specific points of a game that requires more performance makes RVC to increase the delay of the output. A friend have a 3080 (not Ti) and she runs into choppy cuts and stutter when on similar scenarios. So if you want to use RVC without much issues, there is actually not "too much" GPU option. VRAM it's very important too so get the 16GB model.
> 
> Also consider that in a couple of months nvidia will launch the 5000 series. While it will be just the highest top ends and not the mid range ones, I am sure that it will also make every other card to drop in price. So you may want to hold a little bit.



> Hello, do you have any recommendations for me to choose one of the following graphics cards
> 
> 1. Rtx 3060 [Msi Gaming X](342 USD)
> 2. Rtx 4060 ti [Msi Gaming X] 8gb ver (435-491 USD)
> 3. Rtx 4060 ti [Msi Gaming X] 16gb ver (560 USD)
>    Note:IDR to USD
> 
> If there is no significant delay, voice input and output, and no glitching or chopped voice between 3060 and 4060ti 8gb, I will buy 3060. However, I need advice from you if my choice is right or not.
> 
> By the way, here are my PC specs: CPU: Ryzen 5 5600G (stock Cooler) Motherboard: Asus TUF Gaming B550 plus wifi II RAM: t-force delta 16gb ddr4 3200mhz cl16 (dual channel) Ssd: Kingston Kc3000 1TB Psu: Cooler master g600 80+ gold



> There will be differences because the percentage of the GPU used won't be the same between these cards. Usually you aim for the one with the most cuda cores so, in this case, the 4060Ti may be the best option. But keep in mind that even then it may not be enough to run it within certain demanding games (if that's what you aim to do). I have a 3080Ti and there are cases where certain games or specific points of a game that requires more performance makes RVC to increase the delay of the output. A friend have a 3080 (not Ti) and she runs into choppy cuts and stutter when on similar scenarios. So if you want to use RVC without much issues, there is actually not "too much" GPU option. VRAM it's very important too so get the 16GB model.
> 
> Also consider that in a couple of months nvidia will launch the 5000 series. While it will be just the highest top ends and not the mid range ones, I am sure that it will also make every other card to drop in price. So you may want to hold a little bit.

So aku perlu menunggu hingga Nvidia merilis seri 50 nya supaya harga seri terdahulunya turun?

**Comment by KuukoShan** [comment]
3060 may not be enough unless you don't want to use it within a game and only for the voice with Discord. It also has too little VRAM 

The CPU does matter as well since certain settings of the software uses the CPU.

For example, if you plan on using it with vrchat, which is a very cpu intensive game, then be ready to have a quite good and modern CPU or you won't be able to play while RVC it's running.

You guys need to keep in mind that RVC can be very intensive specially when used together with other softwares or games.

**Comment by Huuan007** [comment]
da cores so, in this case, the 4060Ti may be the best option. But keep in mind that even then it may not be enough to run it within certain demanding games (if that's what you aim to do). I have a 3080Ti and there are cases where certain games or specific points of a game that requires more performance makes RVC to increase the delay of the output. A friend have a 3080 (not Ti) and she runs into choppy cuts and stutter when on similar scenarios. So if you want to use RVC without much issues, there is actually not "too much" GPU option. VRAM it's very important too so get the 16GB model.
> 
> Also consider that in a couple of months nvidia will launch the 5000 series. While it will be just the highest top ends and not the mid range ones, I am sure that it will also make every other card to drop in price. So you may want to hold a little bit.

So I just need to wait until Nvidia releases the 50 series so that the previous series drops in price and I can buy the RTX 4060 Ti 16GB at a cheaper price?

**Comment by Huuan007** [comment]
So I just need to wait until Nvidia releases the 50 series so that the previous series drops in price and I can buy the RTX 4060 Ti 16GB at a cheaper price?

**Comment by Huuan007** [comment]
Btw, are rtx 4070 is good too?

**Comment by KuukoShan** [comment]
Yeah. Price drop isn't a warranty but it should happen. If a 5090 and 5080 are released, their 4000 equivalent should drop in price and the whole range would come down as well as 4080 could be closer in price to the 4070.

And yes the 4070 it's very good regardless of the amount of vram. The quality of the voices are not so much tied to the hardware but yo the tasks that you want to do among voice changing. So a 3060 will have the same quality as a 4090 under the same settings but a 4090 will have way more room to run games and other tasks without slowing down the voice changer. The quality of the voice relies more in the models itself.

**Comment by Huuan007** [comment]
> Yeah. Price drop isn't a warranty but it should happen. If a 5090 and 5080 are released, their 4000 equivalent should drop in price and the whole range would come down as well as 4080 could be closer in price to the 4070.
> 
> And yes the 4070 it's very good regardless of the amount of vram. The quality of the voices are not so much tied to the hardware but to the tasks that you want to do among voice changing. So a 3060 will have the same quality as a 4090 under the same settings but a 4090 will have way more room to run games and other tasks without slowing down the voice changer. The quality of the voice relies more in the models itself.

Thank you for answering my question.  By the way, what type of mic that good for the RVC?  Also, Can you tell me the name/brand of your mic gear? :)

**Comment by Huuan007** [comment]
(My budget for mic is 40 dollar)

**Comment by KuukoShan** [comment]
This is a voice changer, not a voice morpher, so the quality of the mic shouldn't have a huge impact if any. As long as what you say can be understood clearly enough, it will work fine. You won't get better results using an expensive professional mic. In fact, the better the mic the more prone to any tiny ambient sound which will result in the voice changer trying to "speak" that sound. I personally use the mic from the headset. It's a Logitech Pro X Wireless but you do not need anything that expensive. If you already have a headset with mic, it will do the job just fine.

**Comment by Huuan007** [comment]
> This is a voice changer, not a voice morpher, so the quality of the mic shouldn't have a huge impact if any. As long as what you say can be understood clearly enough, it will work fine. You won't get better results using an expensive professional mic. In fact, the better the mic the more prone to any tiny ambient sound which will result in the voice changer trying to "speak" that sound. I personally use the mic from the headset. It's a Logitech Pro X Wireless but you do not need anything that expensive. If you already have a headset with mic, it will do the job just fine.

Can I ask you what brand should I choose?
 In the local online shop where I live, I found a Zotac Twin Edge RTX 4070 Super for only 625 USD (9.9 million IDR)
 But many people think Zotac uses cheaper components to reduce prices and production costs.

 So, is the choice to choose the Zotac RTX 4070 Super good or am I better off choosing the Asus Pro art OC RTX 4060 ti 16Gb (607 USD) because the brand and components are better?

 Or should I just choose the Zotac RTX 4060 Ti 16 GB AMP because the price is only (445 USD)?

Btw, what chunk and extras settings that I will use if I choose the 4060 ti 16 gb or other GPU I asked

**Comment by KuukoShan** [comment]
Brands and quality it's a complex subject. I tend to choose Asus but that is not warranty that it can't break regardless of the brand. I would choose the 4070 super since it has a very high performance. I guess your country have some warranty period so you shouldn't worry too much. Also, it has a really good price. That's up to you.

About chunk and extra size. The chunk will give you a direct delay so the more delay the better quality. This is regardless of the GPU that you use. Here you have to choose if you compromise quality in favor of speed or not. I personally have it set to 128 with an extra of 16K.
Thw chunk size will directly affect the quality. If you speak and the voice stutter or cuts or does weird things at the end of the words, increase this. The extra setting is how much info of what you have already spoke will be used to determine the exit of the voice so the higher you set this, the more past data will use for the next time that you speak. This doesn't increase delay but increases gpu usage. It's easy to increase this one without expecting major slowdowns at all. But all these settings also comes down to each voice model. Some sounds bad regardless of the settings while some sounds really good without increasing the quality much. You will have to test and adjust but the mentioned base settings are a good starting point for good quality without too much delay.

**Comment by Huuan007** [comment]
> Brands and quality it's a complex subject. I tend to choose Asus but that is not warranty that it can't break regardless of the brand. I would choose the 4070 super since it has a very high performance. I guess your country have some warranty period so you shouldn't worry too much. Also, it has a really good price. That's up to you.
> 
> About chunk and extra size. The chunk will give you a direct delay so the more delay the better quality. This is regardless of the GPU that you use. Here you have to choose if you compromise quality in favor of speed or not. I personally have it set to 128 with an extra of 16K. Thw chunk size will directly affect the quality. If you speak and the voice stutter or cuts or does weird things at the end of the words, increase this. The extra setting is how much info of what you have already spoke will be used to determine the exit of the voice so the higher you set this, the more past data will use for the next time that you speak. This doesn't increase delay but increases gpu usage. It's easy to increase this one without expecting major slowdowns at all. But all these settings also comes down to each voice model. Some sounds bad regardless of the settings while some sounds really good without increasing the quality much. You will have to test and adjust but the mentioned base settings are a good starting point for good quality without too much delay.




Hello, im still choosing my GPU but i think i will take the zotac gpus because of lower prices than the other

Zotac AMP Rtx 4060 ti 16gb (425$)from (445$) 

Zotac Rtx 4070 (549$) 

Zotac Rtx 4070 Super (595$) 

I think the 4070 and the super one still at msrp prices

And Well, im just playing genshin, Honkai StarRail, honkai impact, War Thunder, and valorant
But i want to use Rvc while playing with my friend on coop mode genshin and valorant via discord or in game voice. 

Idk if the 4070 super is too much or it can be future proof because this will be my first gpu that if try new game there no problem

Here my discord if you want to give me advice: ahuman7788

**Comment by KuukoShan** [comment]
There is no such thing as "too much" when you want to use a GPU with RVC. Keep in mind that a lot of people goes for a 4090 to be able to play without issues while using RVC. 4070 Super is a great option, specially because it's just few bucks of difference with the non super model. It should be fine to play games while using RVC. Depending on how much demanding the game is, it may cause stutter or delay on RVC, but that's kind of expected and should happen much. It also depends on your target framerate so keeping 144fps with RVC can be challenging in some games. Go for the super, it's well priced and it's a good gpu. Future proof? Nowadays games jump on quality quite a lot and the tech is moving fast enough to make just a prior gen obsolete. Like, for example, I can't use nvidia DLSS 3 with frame generation on my 3080Ti. So it's hard to know how long a card will work. That said, it doesn't means that you can't just disable few super new features and keen playing so do not worry too much about it.

But I have something that it's quite important right now. In a matter of 3 or 4 months, 5000 series will release. Probable 5080 and 5090. That means that prior series should see a drop in price. And little later 5070 and 5060 will happen too. And, while I know that about "we can't wait forever for future better hardware", it's true that right now it's probably the worst moment to update since nvidia release new gpus every 2 years or so and the new ones are really around the corner.
If you really crave for it and your actual one can't meet what you need to play and use RVC then go for the 4070S. Otherwise patience may reward you because you may be able to buy that same 4070S way cheaper or even a 4080 for the actual price of the 4070S.

**Comment by zerishachan-afk** [comment]
> There is no such thing as "too much" when you want to use a GPU with RVC. Keep in mind that a lot of people goes for a 4090 to be able to play without issues while using RVC. 4070 Super is a great option, specially because it's just few bucks of difference with the non super model. It should be fine to play games while using RVC. Depending on how much demanding the game is, it may cause stutter or delay on RVC, but that's kind of expected and shouldn't happen much. It also depends on your target framerate so keeping 144fps with RVC can be challenging in some games. Go for the super, it's well priced and it's a good gpu. Future proof? Nowadays games jump on quality quite a lot and the tech is moving fast enough to make just a prior gen obsolete. Like, for example, I can't use nvidia DLSS 3 with frame generation on my 3080Ti. So it's hard to know how long a card will work. That said, it doesn't means that you can't just disable few super new features and keen playing so do not worry too much about it.
> 
> But I have something that it's quite important right now. In a matter of 3 or 4 months, 5000 series will release. Probable 5080 and 5090. That means that prior series should see a drop in price. And little later 5070 and 5060 will happen too. And, while I know that about "we can't wait forever for future better hardware", it's true that right now it's probably the worst moment to update since nvidia release new gpus every 2 years or so and the new ones are really around the corner. If you really crave for it and your actual one can't meet what you need to play and use RVC then go for the 4070S. Otherwise patience may reward you because you may be able to buy that same 4070S way cheaper or even a 4080 for the actual price of the 4070S.



what is your GPU anyway? did you use Rtx 3060, 4060 or 4090 for now??

**Comment by Kuuko-fokkusugaru** [comment]
@zerishachan-afk  at the time of writing that it was a 3080Ti. It was enough to play Minecraft with PTGI shader at 1440p 60fps and leave some room for RVC. It was also fine to play Fortnite maxed at 120fps as long as nanite, virtual shadows, and GI was disabled (otherwise RVC would stutter and delay a lot). Also tested playing The Forest maxed out at 90fps if I remember correctly (though, for some reason, in the transition between night and day, a huge delay would happen for some seconds due probably everything being respawned and recalculated in the game). That should give you a rough idea of the performance.
Right now I use a 4090 so it's less likely to get stutter or delay. I still play Fortnite with the same settings because that gives me 180fps and plenty of room for RVC. Also, playing vrchat with the 3080Ti would give me huge delays of about 12 seconds and more and a lot of stutter and glitches. With the 4090 I can use vrchat without any issue at all even if the max amount of fps isn't achieved on every world. I hope that helps.

**Comment by zerishachan-afk** [comment]
> [@zerishachan-afk](https://github.com/zerishachan-afk) at the time of writing that it was a 3080Ti. It was enough to play Minecraft with PTGI shader at 1440p 60fps and leave some room for RVC. It was also fine to play Fortnite maxed at 120fps as long as nanite, virtual shadows, and GI was disabled (otherwise RVC would stutter and delay a lot). Also tested playing The Forest maxed out at 90fps if I remember correctly (though, for some reason, in the transition between night and day, a huge delay would happen for some seconds due probably everything being respawned and recalculated in the game). That should give you a rough idea of the performance. Right now I use a 4090 so it's less likely to get stutter or delay. I still play Fortnite with the same settings because that gives me 180fps and plenty of room for RVC. Also, playing vrchat with the 3080Ti would give me huge delays of about 12 seconds and more and a lot of stutter and glitches. With the 4090 I can use vrchat without any issue at all even if the max amount of fps isn't achieved on every world. I hope that helps.


i wanna to ask, did yoy think, if my Rtx 4060 can enough for run Okada RVC with 128 Chunk / 192 Chunk with an Extra 16.000 (Just For Discord and Game Yugioh Master Duel) ?? @Kuuko-fokkusugaru , anyway i have CPU 8700F

**Comment by Kuuko-fokkusugaru** [comment]
Yeah, definitely it should be able to. Anyway, just try? It's not like you have to pay for the software, so just download and test.
